documentation/Unity-WebGL-NodeJS-Server-Setup.md

```diff
@@ -0,0 +1,13 @@
+The following steps can be used to host the WebGL content via Node JS server.<br>
+This is an alternative to serving the built Unity WebGL files over an Apache server whose instructions can be found here: [Apache Server Setup](https://github.com/eknuviad/domain-model-assistant/blob/main/documentation/Unity-WebGL-Apache-Server-Setup.md)<br>
+NB: *This was tested on Windows 10. Steps may vary for MAC OS* <br>
```

Comment:
Please also add the caveat that this is not officially supported by Unity and that more testing is needed to make sure the app actually works. I have no reason to think it won't, but it's good to be extra careful with things we are not too familiar with.